apps/aws-privatelink.mdx

```diff
@@ -4,28 +4,30 @@ description: 'This page explains how to enrich your Axiom experience with AWS Pr
 
-## Setup
-
-1. Connect the AWS Console to region `us-east-1` and create a VPC. For more information, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/privatelink/getting-started.html).
-1. Start creating a VPC endpoint. For more information, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html#create-interface-endpoint-aws).
-1. In **Service category**, select **Other endpoint services**.
-1. In **Service name**, enter `com.amazonaws.vpce.us-east-1.vpce-svc-05a64735cdf68866b` to establish AWS PrivateLink for `api.axiom.co`.
-1. Click **Verify service**. If this doesn’t succeed, reach out to [Axiom Support](https://axiom.co/contact).
-1. Select the VPC and subnets that you want to connect to the Axiom VPC service endpoint. Ensure that **Enable DNS name** is turned on and the security group accepts inbound traffic on TCP port `443`.
-1. Finish the setup and wait for the VPC endpoint to become available. This usually takes 10 minutes.
+US regions:
+- `us-east-1`
+- `us-east-2`
+- `us-west-1`
+- `us-west-2`
 
-## Configure Amazon VPC Peering
+EU regions:
+- `eu-west-1`
+- `eu-west-2`
+- `eu-west-3`
+- `eu-central-1`
 
-To route traffic to Axiom’s PrivateLink offering in `us-east-1` from other AWS regions, use inter-region [Amazon VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html). Inter-region VPC peering allows you to establish connections between VPCs across different AWS regions. This allows VPC resources in different regions to communicate with each other using private IP addresses.
+To connect to a region that isn’t listed above, [contact Axiom](https://axiom.co/contact).
 
-After following the [setup in `us-east-1`](#setup), configure VPC peering to make the PrivateLink endpoint available in another region to send logs to Axiom over PrivateLink. For more information, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/working-with-vpc-peering.html).
+## Setup
 
-When configuring PrivateLink with VPC peering, Amazon Route 53 is useful for resolving private DNS hostnames within your VPCs. Amazon Route 53 allows you to create private hosted zones within your VPC. These private hosted zones allow you to use custom domain names for your resources, such as EC2 instances, ELB load balancers, or RDS instances, without exposing them to the public internet. For more information, see the [AWS documentation](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html).
\ No newline at end of file
+1. In your VPC Console, go to **PrivateLink and Lattice > Endpoints**, and then click **Create endpoint**.
+1. Select **PrivateLink ready partner services**, and then enter `com.amazonaws.vpce.us-east-1.vpce-svc-05a64735cdf68866b` as the service name.
+1. Under **Service Region**, turn on **Cross-region endpoint**, and then click **Verify service**.
```

Comment:
When he turn on **Cross-region endpoint**, he will need to select a region, the region should be `us-east-1` where the privatelink is setup on our side